examples/overplotFiberTrace.ipynb

```diff
@@ -96,9 +98,53 @@
     "            points = [pointA, pointB]\n",
     "            display.line(points, ctype='red')"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "collapsed": true
+   },
```

Comment:
You should remove all outputs before saving and committing notebooks (unless you really want people to be able to see the notebooks without running them, which isn't true here as you're not inlining the image displays [which you can't do with ds9, but maybe you're using a different display backend])